src/api/types.ts

```diff
@@ -179,13 +179,15 @@ interface DeleteApplicationByReferenceNumberVariables {
   referenceNumber: number;
 }
 
-interface MappedCompaniesHouseAddress {
+interface Address {
   careOf?: string;
   premises?: string;
   addressLine1?: string;
   addressLine2?: string;
   locality?: string;
+  town?: string;
   region?: string;
+  county?: string;
```

Comment:
What are the differences between these?:

```js
locality
town
region
county
```
Seems 2 of them might have the same meaning and we could map instead